dbux-graph-host/src/graph/controllers/ContextNodeManager.js

```diff
@@ -9,14 +9,55 @@ import HostComponentEndpoint from '../../componentLib/HostComponentEndpoint';
 // eslint-disable-next-line no-unused-vars
 const { log, debug, warn, error: logError } = newLogger('ContextNodeManager');
 
-let SelectorType = {
+const SelectorTypeConfig = {
   ObjectTrace: 1,
   StaticContext: 2,
   SearchContext: 3,
-  SearchTrace: 4
+  SearchTrace: 4,
+  SearchValue: 5,
+};
+
+/**
+ * @type {(Enum|typeof SelectorTypeConfig)}
+ */
+const SelectorType = new Enum(SelectorTypeConfig);
+
+const FindContextsByMode = {
+  [SelectorType.ObjectTrace]: (selector) => {
+    const { applicationId, traceId: originTraceId } = selector;
+    const dp = allApplications.getById(applicationId).dataProvider;
+
+    const { traceId } = dp.util.getValueTrace(originTraceId);
+    const refId = dp.util.getTraceRefId(traceId);
+    const contexts = dp.util.getContextsByRefId(refId);
+    return contexts;
+  },
+  [SelectorType.StaticContext]: (selector) => {
+    const { applicationId, staticContextId } = selector;
```

Comment:
@MiccWan Why is this only searching a singular application, and the others search multiple?

Reply:
We used this to highlight `Contexts` of same `StaticContext` in `CallGraph` (with a small button in `ContextNode`). The search algorithm only takes `staticContextId` into account, which is meaningless for the other applications.